arangodb-net-standard/CursorApi/CursorApiClient.cs

```diff
@@ -98,7 +98,8 @@ public async Task<DeleteCursorResponse> DeleteCursorAsync(string cursorId)
 
         public async Task<PutCursorResponse<T>> PutCursorAsync<T>(string cursorId)
         {
-            using (var response = await _client.PutAsync(_cursorApiPath + "/" + WebUtility.UrlEncode(cursorId), new StringContent(string.Empty)))
+            var uriString = _cursorApiPath + "/" + WebUtility.UrlEncode(cursorId);
```

Comment:
Just a trivial change: I think we can use `string` directly here? The `var` keywork doesn't provide much information, so this could be changed to `string uri` which makes it more redeable in my opinion.